carmin-server/server/test/resources/test_execution.py

```diff
@@ -0,0 +1,41 @@
+import json
+import pytest
+
+from server.resources.models.execution import ExecutionSchema, Execution
+from server.common.error_codes_and_messages import EXECUTION_NOT_FOUND
+from server.test.fakedata.executions import POST_VALID_EXECUTION
+from server.test.resources.test_executions import test_config
+from server.test.fakedata.users import standard_user
+from server.test.utils import load_json_data, error_from_response
+
+
+@pytest.fixture
+def execution_id(test_config) -> str:
+    response = test_config.test_client.post(
```

Comment:
Let's have a talk about this tomorrow please. I understand why we use the test_client to do our test setup, but using the app code to perform setup functions is not recommended regarding testing.

Reply:
Due to the complex setup required and as we already have exhaustive testing regarding execution POST, this is correct.